data/tlg2042/tlg057/tlg2042.tlg057.1st1K-grc1.xml

```diff
@@ -0,0 +1,474 @@
+                                <date>1857</date>
+                            </imprint>
+                        </monogr>
+                        <series>
+                            <title level="s" xml:lang="lat">Patrologiae cursus completus (series
+                                Graeca)</title>
+                            <biblScope unit="volume">12</biblScope>
+                        </series>
+                        <ref
+                            target="https://archive.org/stream/patrologiae_cursus_completus_gr_vol_012#page/n519/mode/2up"
+                            >Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+)" n="section" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="lat">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg2042.tlg057.1st1K-grc1">
+                        <pb n="1032"/>
+                        <head>ΩΡΙΓΕΝΟΥΣ ΕΚΛΟΓΑΙ ΕΙΣ ΙΩΒ.</head>
+
+                        <div type="textpart" subtype="section" n="1">
+                            <head>ΚΕΦ. Α΄.</head>
+                            <p>Οἱ ἱππεῖς ἐποίησαν ἡμῖν κεφαλὰς τρεῖς καὶ ἐκύκλωσαν τὰς καμήλους. Τὰς
+                                μὲν οὖν καμήλους πεποίηκε γενέσθαι ὑπὸ τοὺς ἱππεῖς· τὰς ὄνους δὲ καὶ
+                                τὰ ζεύγη τῶν βοῶν ὑπὸ τοὺς αἰχμαλωτίζοντας· τὰ πρόβατα δὲ ἐμίσει·
+                                διὰ τοῦτο πῦρ ἐπʼ αὐτὰ κατήγαγεν· ἀλλὰ καὶ τοῖς υἱοῖς ἐπεβούλευε,
+                                καὶ καιρὸν ἐζήτει τοῦ αὐτοὺς ἀπολέσαι. Εἰ εὑρήκει ἀναγινώσκοντας,
+                                οὐκ ἂν ἥψατο τῆς οἰκίας, οὐδὲ τόπον εἶχε τοῦ αὐτοὺς ἀποκτεῖναι· εἰ
+                                εὑρήκει προσευχομένους, οὐδὲν ἐδύνατο ποιῆσαι κατʼ αὐτῶν· ἀλλʼ ὅτε
+                                εὖρεν εὐκαιρίαν, ἴσχυσε. Ποῖος ἦν ὁ καιρός; μέθη ἦν, συμπόσιον ἦν.
+                                Τῷ δὲ τῶν γωνιῶν ἅψασθαι τὸ πνεῦμα, ἐπίστησον, ὅτι ἐν ταῖς τῶν
+                                γωνιῶν πλατείαις (16·) οἱ ὑποκριταὶ ἑστῶτες προσεύχονται· πᾶς δὲ ὁ
+                                μὴ ἐπιβεβηκὼς εὐθείας ὀδοῦ ταῖς γωνίαις κέχρηται· πλατεῖα δὲ γωνία
+                                ἐστὶν ἡ πολλὴ καὶ ἀνειμένη κακία.</p>
+                        </div>
+
+                        <div type="textpart" subtype="section" n="2">
```

Comment:
Hmm, looking at the section numbers: the sections start with a Greek number representing, I think, the actual chapter in Job that the comment comes from. So this one is actually "19" instead of "2". I think it would be best to use those for the section numbers instead of numbering them sequentially. You can check this website to convert the Greek letters into numbers: http://www.russellcottrell.com/greek/utilities/greeknumberconverter.htm.